clang/lib/Parse/ParseDecl.cpp

```diff
@@ -5828,6 +5832,7 @@ bool Parser::isKnownToBeTypeSpecifier(const Token &Tok) const {
   case tok::kw__ExtInt:
   case tok::kw__BitInt:
   case tok::kw___bf16:
+  case tok::kw___mfp8:
```

Comment:
Do you know why this and the similar changes are needed?  A quick search yielded nothing similar for the SVE type names, so just wondering if we're missing some auto-generation somewhere.